### ISSUE TITLE
Address core-command Blackbox Test Failure

### ISSUE THREAD

**Issue by michaelestrin** [edgexfoundry/edgex-go#2237]
From https://jenkins.edgexfoundry.org/view/blackbox-testing/job/blackbox-testing-centos7-blackbox-4c-2g-master/682/consoleFull:

```
07:02:44 core-command
07:02:44 
07:02:44 ❏ device
07:02:44 ↳ http://localhost:48082/api/v1/device
07:02:45   GET http://edgex-core-command:48082/api/v1/device [500 Internal Server Error, 206B, 243ms]
07:02:45   1. Status code is 200
07:02:45   ✓  Response time is less than 800
07:02:45 
07:02:45 ↳ 17 http://localhost:48082/api/v1/device/:id
07:02:45   GET http://edgex-core-command:48082/api/v1/device/name/Livingroom automation [200 OK, 764B, 23ms]
07:02:45   GET http://edgex-core-command:48082/api/v1/device/e330d5dd-414c-4fd6-8a30-9ea4f5c62a71 [200 OK, 764B, 25ms]
07:02:45   ┌
07:02:45   │ 'Validation failed: ', null
07:02:45   └
07:02:45   ✓  Status code is 200
07:02:45   ✓  Response time is less than 1200 ms
07:02:45   ✓  Has Content-Type
07:02:45   ✓  Content-Type is application/json
07:02:45   ✓  Is Valid Response Schema
07:02:45 
07:02:45 ↳ 20_200 http://localhost:48082/api/v1/device/:id/command/{commandId}
07:02:45   GET http://edgex-core-command:48082/api/v1/device/name/Random-Boolean-Device [200 OK, 1002B, 11ms]
07:02:45   GET http://edgex-core-command:48082/api/v1/device/47e15482-e0f6-4449-8e89-6d8d1d51bd8a/command/46284680-ed64-4fea-aac9-09878d46ebd7 [200 OK, 299B, 61ms]
07:02:45   ┌
07:02:45   │ 'Validation failed: ', null
07:02:45   └
07:02:45   ✓  Status code is 200
07:02:45   ✓  Response time is less than 800ms
07:02:45   ✓  Has Content-Type
07:02:45   ✓  Content-Type is application/json
07:02:45   ✓  Is Valid Response Schema
07:02:45 
07:02:45 ↳ 21_200 http://localhost:48082/api/v1/device/:id/command/{commandId}
07:02:45   GET http://edgex-core-command:48082/api/v1/device/name/Random-Integer-Device [200 OK, 3.1KB, 33ms]
07:02:45   PUT http://edgex-core-command:48082/api/v1/device/b42796cb-fc2a-4091-8fc9-ce8112895e36/command/c7e48377-247f-4425-941b-b43bc98ea0df [200 OK, 107B, 83ms]
07:02:45   ✓  Status code is 200
07:02:45   ✓  Response time is less than 800ms
07:02:45 
07:02:45 ↳ http://localhost:48082/api/v1/device/name/:name
07:02:45   GET http://edgex-core-command:48082/api/v1/device/name/Livingroom automation [200 OK, 764B, 12ms]
07:02:45   ┌
07:02:45   │ 'Validation failed: ', null
07:02:45   └
07:02:45   ✓  Status code is 200
07:02:45   ✓  Response time is less than 1200 ms
07:02:45   ✓  Has Content-Type
07:02:45   ✓  Content-Type is application/json
07:02:45   ✓  Is Valid Response Schema
07:02:45 
07:02:45 ↳ 30_200 http://localhost:48082/api/v1/device/name/{name}/command/{commandname}
07:02:45   GET http://edgex-core-command:48082/api/v1/device/name/Random-Integer-Device [200 OK, 3.1KB, 22ms]
07:02:45   GET http://edgex-core-command:48082/api/v1/device/name/Random-Integer-Device/command/Int8 [200 OK, 296B, 22ms]
07:02:45   ✓  Status code is 200
07:02:45   ✓  Response time is less than 800
07:02:45   ✓  Has Content-Type
07:02:45   ✓  Content-Type is application/json
07:02:45 
07:02:45 ↳ 31_200 http://localhost:48082/api/v1/device/name/{name}/command/{commandname}
07:02:45   GET http://edgex-core-command:48082/api/v1/device/name/Random-Integer-Device [200 OK, 3.1KB, 25ms]
07:02:45   PUT http://edgex-core-command:48082/api/v1/device/name/Random-Integer-Device/command/Int8 [200 OK, 107B, 52ms]
07:02:45   ✓  Status code is 200
07:02:45   ✓  Response time is less than 800
07:02:46 
07:02:46 ┌─────────────────────────┬───────────────────┬───────────────────┐
07:02:46 │                         │          executed │            failed │
07:02:46 ├─────────────────────────┼───────────────────┼───────────────────┤
07:02:46 │              iterations │                 1 │                 0 │
07:02:46 ├─────────────────────────┼───────────────────┼───────────────────┤
07:02:46 │                requests │                12 │                 0 │
07:02:46 ├─────────────────────────┼───────────────────┼───────────────────┤
07:02:46 │            test-scripts │                 7 │                 0 │
07:02:46 ├─────────────────────────┼───────────────────┼───────────────────┤
07:02:46 │      prerequest-scripts │                 5 │                 0 │
07:02:46 ├─────────────────────────┼───────────────────┼───────────────────┤
07:02:46 │              assertions │                25 │                 1 │
07:02:46 ├─────────────────────────┴───────────────────┴───────────────────┤
07:02:46 │ total run duration: 1003ms                                      │
07:02:46 ├─────────────────────────────────────────────────────────────────┤
07:02:46 │ total data received: 12.16KB (approx)                           │
07:02:46 ├─────────────────────────────────────────────────────────────────┤
07:02:46 │ average response time: 51ms [min: 11ms, max: 243ms, s.d.: 61ms] │
07:02:46 └─────────────────────────────────────────────────────────────────┘
07:02:46 
07:02:46   #  failure         detail                                                 
07:02:46                                                                             
07:02:46  1.  AssertionError  Status code is 200                                     
07:02:46                      expected false to be truthy                            
07:02:46                      at assertion:0 in test-script                          
07:02:46                      inside "device / http://localhost:48082/api/v1/device" 
```

**Comment by brandonforster** [comment]
Fixed by https://github.com/edgexfoundry/go-mod-core-contracts/pull/195

**Comment by cloudxxx8** [comment]
@brandonforster @michaelestrin please reopen this issue the failed test case is still on the latest black box test report.
This issue is fixed in go-mod-core-contracts **v0.1.41**, but the current edgex-go is still using **v0.1.37**
github.com/edgexfoundry/go-mod-core-contracts v0.1.37
